cmd/krew/cmd/update.go

```diff
@@ -132,6 +149,9 @@ func ensureIndexUpdated(_ *cobra.Command, _ []string) error {
 	// TODO(chriskim06) consider commenting this out when refactoring for custom indexes
 	showUpdatedPlugins(os.Stderr, preUpdateIndex, posUpdateIndex, installedPlugins)
 
```

Comment:
I’ve realized this if (+counting) isnt necessary since errors.Wrap already handles nil err. Maybe remove?

Reply:
Sure I can do that. Do you think it will look weird if this ends with the `return errors.Wrapf(...)`?

Reply:
I think it’s fine. I just realized the check is redundant.

Reply:
Ok I'll remove the check then.